### PR TITLE
Remove duped docs and update instructions

### PR DESCRIPTION
**Pull request by jyee**
### What does this PR do?

This PR removes histogram percentiles and APM documentation that was not specific to the buildpack (and was out of sync with those dedicated docs). It also updates information for the 1.2.0 buildpack release.

### Motivation
See https://trello.com/c/luWTf84C/2430-apm-docs-on-heroku for doc removal
See https://datadog.zendesk.com/agent/tickets/134063 for metrics aggregation issue.

### CODE REVIEW

**Comment by jyee** [comment]
Preview is a bit borked because the code wrap issue and footnote style links are broken in tables. But Guac is working on fixing that (currently affects prod too).

File: content/agent/basic_agent_usage/heroku.md [hunk 1]
```diff
@@ -41,66 +43,22 @@ In addition to the environment variables shown above, there are a number of othe
...
-ies
-Datadog.configure do |c|
-  c.use :rails, service_name: ENV['DD_SERVICE_NAME'] || 'my-app'
-end
-```
-
-Setting the service name varies according to your language or supported framework. Reference the [Datadog libraries list][6] for specific language support.
+## More information
 
+For more information, view the source code, or contribute to this project, refer to the [Github project page][6].
```
Comment (AndrewMcBurney) [inline content/agent/basic_agent_usage/heroku.md]: Could you please break this up into two sentences.

> For more information, view the source code or contribute to this project. Refer to the [Github project page][6].

Comment (AndrewMcBurney) [reply content/agent/basic_agent_usage/heroku.md]: Hmm, I don't like my suggested phrasing, but could this sentence be broken up still, or rephrased?

Comment (jyee) [reply content/agent/basic_agent_usage/heroku.md]: How about "Visit the Github project page for more information and to view the source code"?

I'm realizing that it's likely assumed that going to github is where contributing info would be, so maybe it doesnt need to be said.

Comment (AndrewMcBurney) [reply content/agent/basic_agent_usage/heroku.md]: I like that a lot!

**Pull request merged**